### PR TITLE
Add back button everywhere

### PR DESCRIPTION
**Pull request by jackielincroft**
## Why
https://app.clickup.com/t/1neyd1z

## This PR
Added back button component that already existed on "Your Profile" page to all other pages

## Screenshots
Screen Shot 2021-10-14 at 6 05 05 PM
Screen Shot 2021-10-14 at 6 12 13 PM
Screen Shot 2021-10-14 at 6 12 40 PM
Screen Shot 2021-10-14 at 6 13 26 PM

### CODE REVIEW

**Comment by granttebeau** [comment]
@jackielincroft have you addressed the above comments yet?

File: src/containers/selectSchool/SelectSchool.tsx [hunk 1]
```diff
@@ -72,55 +74,60 @@ const SelectSchool: React.FC = () => {
...
-              <FormButtons.Button
-                text="Next"
-                type="primary"
-                isSubmit
-                disabled={submitDisabled}
-              />
-            </FormButtons>
-          </Form>
-        </FormContentContainer>
+        <Container>
```
[the remark this answers is not in the document]
Comment (jackielincroft) [reply src/containers/selectSchool/SelectSchool.tsx:77]: IMO it looks fine without the container, it's just that the back button is not all the way in the top left like the other pages, so I was trying to maintain consistency.

Screen Shot 2021-11-07 at 12 05 49 PM

**Comment by jackielincroft** [comment]
> @jackielincroft have you addressed the above comments yet?

Sorry for the delay, yep just refactored back button into its own component!

**Review by annettastogniew57 — APPROVED** [review]
lgtm

**Review by rymaju — APPROVED** [review]
LGTM

**Pull request merged**